File: moderate/email_regex.lua
regex1 = "^[%w.+-=]+@%w+[%w.]*%w*%.%w%w%w?%w?$"
regex2 = "^\"[%w@.+-=]+\"@%w+[%w.]*%w*%.%w%w%w?%w?$"

for line in io.lines(arg[1]) do
  print((line:find(regex1) or line:find(regex2)) and "true" or "false")
end

File: moderate/email_regex.clj
(with-open [rdr (clojure.java.io/reader (first *command-line-args*))]
  (doseq [line (line-seq rdr)]
    (println
      (if (re-find #"^((\"[0-9A-Za-z@.+-=]+\")|([0-9A-Za-z.+-=]+))@\w*(\w+\.)+\w{2,4}$" line)
        "true"
        "false"))))
